engine/src/conversion/codegen_cpp/mod.rs

```diff
@@ -235,7 +235,8 @@ impl<'a> CppCodeGenerator<'a> {
                 headers, self.inclusions, type_definitions, declarations
             );
             log::info!("Additional C++ decls:\n{}", declarations);
-            let header_name = format!("autocxxgen_{}.h", self.config.get_mod_name());
+            let header_name =
+                self.cpp_codegen_options.header_namer.0(self.config.get_mod_name().to_string());
```

Comment:
Nit: Would you mind adding an `impl` block so we can call a function `.name_header(..)` instead of `.0(..)`

Reply:
Done